include/pstore/mcrepo/compilation.hpp

```diff
@@ -53,11 +53,15 @@ namespace pstore {
     namespace repo {
 
 #define PSTORE_REPO_LINKAGE_TYPES                                                                  \
-    X (append)                                                                                     \
-    X (common)                                                                                     \
-    X (external)                                                                                   \
-    X (internal)                                                                                   \
-    X (linkonce)
+    X (append_linkage)                                                                             \
+    X (common_linkage)                                                                             \
+    X (external_linkage)                                                                           \
+    X (private_linkage)                                                                            \
+    X (internal_linkage)                                                                           \
+    X (linkonceany_linkage)                                                                        \
+    X (linkonceodr_linkage)                                                                        \
+    X (weakany_linkage)                                                                            \
```

Comment:
I‘m sure you get the idea…

Reply:
Yes, I will change: 
1) `weakany_linkage` to `weak_any_linkage`
2) `weakodr_linkage` to `weak_odr_linkage`